examples/three_layer_constant_fluxes.jl

```diff
@@ -392,62 +392,78 @@ tke_budget_statistics = turbulent_kinetic_energy_budget(model, b=b, p=p, U=U, V=
 fields_to_output = merge(model.velocities, model.tracers, (e=e, ϵ=dissipation))
 
 statistics_to_output = merge(primitive_statistics, subfilter_flux_statistics, tke_budget_statistics)
-
-function init_save_some_metadata!(file, model)
-    file["parameters/coriolis_parameter"] = f
-    file["parameters/density"] = 1027.0
-    file["boundary_conditions/θ_top"] = Qᶿ
-    file["boundary_conditions/θ_bottom"] = dθdz_deep
-    file["boundary_conditions/u_top"] = Qᵘ
-    file["boundary_conditions/u_bottom"] = 0.0
-    return nothing
-end
+statistics_to_output = Dict(string(k) => v for (k, v) in statistics_to_output)
 
 @info "Garnishing output writers..."
 
+ρ₀ = 1027
+cₚ = 4000
+
+global_attributes = (
+    name = name,
+    thermocline_type = thermocline_type,
+    reference_density = ρ₀,
+    specific_heat_capacity = cₚ,
+    buoyancy_flux = Qᵇ,
+    momentum_flux = Qᵘ,
+    heat_flux = Qᶿ,
+    heat_flux_Wm⁻² = - ρ₀ * cₚ * Qᶿ,
+    coriolis_parameter = f,
+    thermal_expansion_coefficient = α,
+    gravitational_acceleration = g,
+    boundary_condition_θ_top = Qᶿ,
+    boundary_condition_θ_bottom = dθdz_deep,
+    boundary_condition_u_top = Qᵘ,
+    boundary_condition_u_bottom = 0.0,
+    surface_layer_depth = surface_layer_depth,
+    thermocline_width = thermocline_width,
+    N²_surface_layer = N²_surface_layer,
+    N²_thermocline = N²_thermocline,
+    N²_deep = N²_deep,
+    dθdz_surface_layer = dθdz_surface_layer,
+    dθdz_thermocline = dθdz_thermocline,
+    dθdz_deep = dθdz_deep,
+    θ_surface = θ_surface,
+    θ_transition = θ_transition,
+    θ_deep = θ_deep,
+    z_transition = z_transition,
+    z_deep = z_deep,
+    k_transition = k_transition,
+    k_deep = k_deep
+)
```

Comment:
Is there any way to automate some of this? It's quirky that we already have `model.coriolis.f` but then define an attribute `coriolis_parameter`. Similar for the boundary conditions and gravitational acceleration. How can we automate or standardize saving some of this metadata to ensure that it's recorded as uniformly as possible by all users / LESbrary data generators?

The parameters of the initial condition might be hard to understand out of context, but maybe that's ok.

Reply:
Was thinking this might be useful as an Oceananigans.jl feature in general so I opened an issue: https://github.com/CliMA/Oceananigans.jl/issues/1313

But maybe it makes sense to prototype in LESbrary.jl?

Reply:
If we have a roadmap towards a more permanent solution in Oceananigans, I think its ok to keep a temporary solution / prototype in LESbrary!